exercises/ruby-1-a/.meta/config.json

```diff
@@ -3,32 +3,32 @@
   "test_file": "ruby_1_a_test.rb",
   "tests": [
     {
-      "name": "test_one_word_with_one_vowel",
-      "cmd": "word_with_most_vowels(\"at\")"
+      "name": "test_one_item",
+      "cmd": "ShoppingCart.new.total_amount"
     },
     {
-      "name": "test_one_word_with_one_vowel_and_one_consonant",
-      "cmd": "word_with_most_vowels(\"at\")"
+      "name": "test_initial_total_amount",
+      "cmd": "ShoppingCart.new.add(:D69856).total_amount"
     },
     {
-      "name": "test_one_word_with_two_vowels_and_one_consonant",
-      "cmd": "word_with_most_vowels(\"tea\")"
+      "name": "test_two_items",
+      "cmd": "ShoppingCart.new.add(:D69856).add(:F55690).total_amount"
     },
     {
-      "name": "test_longest_word_is_not_word_with_most_vowels",
-      "cmd": "word_with_most_vowels(\"the area of a circle\")"
+      "name": "test_list_items",
+      "cmd": "ShoppingCart.new.add(:D69856).add(:F55690).list_items"
     },
     {
-      "name": "test_last_word_is_word_with_most_vowels",
-      "cmd": "word_with_most_vowels(\"extremely fresh vegetables\")"
+      "name": "test_items_ordered_alphabetically",
+      "cmd": "ShoppingCart.new.add(:F55690).add(:D69856).list_items"
     },
     {
-      "name": "test_mixed_casing",
-      "cmd": "word_with_most_vowels(\"TEST YOUR MIGHT\")"
+      "name": "test_duplicate_items_sums_price",
+      "cmd": "ShoppingCart.new.add(:F55690).add(:F55690).list_items"
     },
     {
-      "name": "test_ignore_word_without_vowels",
-      "cmd": "word_with_most_vowels(\"great rhythm\")"
+      "name": "test_duplicate_items_uniqs_list",
+      "cmd": "ShoppingCart.new.add(:F55690).add(:F55690).list_items"
```

Comment:
Needs to be updated too

Reply:
Sorry I need to get better at search and replace 😊